test/conformance/router_flavor.go

```diff
@@ -0,0 +1,41 @@
+package conformance
+
+import (
+	"os"
+	"testing"
+)
+
+// RouterFlavor represents the flavor of the Kong router.
+// ref: https://docs.konghq.com/gateway/latest/reference/configuration/#router_flavor
+type RouterFlavor string
+
+const (
+	// RouterFlavorTraditionalCompatible is the traditional compatible router flavor.
+	RouterFlavorTraditionalCompatible RouterFlavor = "traditional_compatible"
+	// RouterFlavorExpressions is the expressions router flavor.
+	RouterFlavorExpressions RouterFlavor = "expressions"
+)
+
+// KongRouterFlavor returns router mode of Kong in tests. Currently supports:
+// - `traditional`
+// - `traditional_compatible`.
+// - `expressions` (experimental, only for testing expression route related tests).
```

Comment:
Do we still consider it experimental?

Reply:
Nope, let me fix that

```suggestion
// - `expressions`
```